src/content/en/progressive-web-apps/checklist.md

```diff
@@ -222,7 +222,7 @@ The following tests are verifiable with the Lighthouse tool.
 <table class="responsive">
   <tbody>
     <tr>
-      <th colspan=2><h3>Pages each have a URL</h3></th>
+      <th colspan=2><h3>Each pages has a URL</h3></th>
```

Comment:
The title was grammatically correct before. The new one isn't. Either change it back or change the new one to 'Each page has'.